distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -4103,6 +4132,9 @@ cd  <DDF_HOME>/etc/exported
 zip exportedFiles.zip *
 ----
 
+//    For below note, if there is a SIMLINK, it will not be included
```

Comment:
This comment can be removed. The reference to DDF-1796 doesn't apply here and the note about symlinks will be added by @bantillo as part of his ticket.